app/src/main/contrib/components/Responsibilities/Overview/List/ResponsibilityComponent.tsx

```diff
@@ -29,8 +29,8 @@ export class ResponsibilityComponent extends React.Component<Props, undefined> {
                         Disease: { mainStore.getDiseaseById(item.scenario.disease).name }
                     </div>
                     <div className={ styles.actions }>
-                        <ButtonLink href={ downloadUrl }>Download input data</ButtonLink>
-                        <button>Upload a new burden estimate set</button>
+                        <ButtonLink href={ downloadUrl }>Download coverage data</ButtonLink>
+                        <button disabled={ true }>Upload a new burden estimate set</button>
```

Comment:
Realised we should disable the bits we haven't implemented yet, before releasing to modellers!